app/models/user.rb

```diff
@@ -27,4 +27,8 @@ class User < ApplicationRecord
   }
   validates :phone_number, uniqueness: true, allow_nil: true
   validates_plausible_phone :phone_number
+
+  def build_profile
+    create_profile(display_name: "#{first_name} #{last_name.capitalize.chr}")
```

Comment:
When either first_name or last_name is updated, changes must be propagated to the profile.